ui/partials/main/thActiveFiltersBar.html

```diff
@@ -1,20 +1,27 @@
 <!-- Show this certain filters are active in Treeherder -->
-<div class="alert-info active-filters-bar">
+<div ng-if="newFieldFilter || filterBarFilters.length"
```

Comment:
So instead of relying on an ominpresent 0px height filter bar, we suppress the div entirely if nothing is happenin'.